cmd/minishift/cmd/start.go

```diff
@@ -181,8 +181,13 @@ func runStart(cmd *cobra.Command, args []string) {
 		preflightChecksBeforeStartingHost()
 	}
 
+	// Get proper OpenShift version
+	requestedOpenShiftVersion, err := getOpenShiftVersion()
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting OpenShift version: %v", err))
+	}
+
```

Comment:
why not put this before L#179 so that you don't need to validate it again for the preflight check?

Reply:
Done :)

Reply:
Why those this need to be in start? This is a preparation step that was moved to the pre-flight... right?